repos/tap/actions/init.js

```diff
@@ -49,8 +48,8 @@ const initTestFile = async (activeFeat, queryFile) => {
 
   const testFile = activeFeat && activeFeat.testPath || queryFile
 
-  // loading example test data from <root>/tests/tests
-  await loadApiFile(exampleFile, (testFile) => upsertActiveRunnerTest(testFile))
+  // load the file tree from root tests folder
+  upsertFileTree(await apiRequest(`/files/get_tree`) || [])
```

Comment:
Same as other comment. Please update the endpoint to be `/files/tree`

Reply:
updated